sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -76,21 +76,21 @@ public async Task TablesTestSetup()
 
             };
 
-            ServiceUri ??= _endpointType switch
+            ServiceUri = _endpointType switch
```

Comment:
@pakrym This was the bug that was leaking variables across sessions - only `ValidateSasCredentials` used this. This would only show up if this test happened to have been recorded against a mismatched set of variables 🤦